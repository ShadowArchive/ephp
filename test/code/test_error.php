<?php

ini_set("html_errors", "On");
no_exist();
