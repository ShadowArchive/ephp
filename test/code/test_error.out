<br/>
<b>Fatal error</b>: Call to undefined function no_exist() in <b>{{CWD}}/test/code/test_error.php</b> on line <b>4</b><br/>
